app/src/containers/RegisterPage/index.js

```diff
@@ -466,9 +486,18 @@ class SignUpPage extends Component {
     this._isMounted = false;
   }
 
+  canBeSubmitted() {
```

Comment:
this is ok, but you can easily combine this in line 772, see below comment, this makes the file unnecessary long